libc/unistd_uids.c

```diff
@@ -0,0 +1,193 @@
+/*
+ * Phoenix-RTOS
+ *
+ * libc-tests
+ *
+ * Testing getuid functions form unistd.h
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Mateusz Niewiadomski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <string.h>
+#include <unistd.h>
+#include <sys/wait.h>
+#include <stdlib.h>
+
+#include <unity_fixture.h>
+
+
+pid_t *child, parent[3], pid;
```

Comment:
Only use-case I see is `child = malloc(sizeof(pid_t) * 6);` anyway
```suggestion
pid_t child[6], parent[3], pid;
```